src/main/resources/db/migration/0200/V226__OrganizationTypes.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE managed_facility_types (
+    id INTEGER NOT NULL PRIMARY KEY,
+    name TEXT NOT NULL
+);
+
+CREATE TABLE organization_types (
+    id INTEGER NOT NULL PRIMARY KEY,
+    name TEXT NOT NULL
+);
+
+CREATE TABLE organization_managed_facility_types (
+  organization_id BIGINT NOT NULL REFERENCES organizations,
+  managed_facility_type_id INTEGER NOT NULL REFERENCES managed_facility_types,
+  UNIQUE (organization_id, managed_facility_type_id)
```

Comment:
Would it be better to make this `PRIMARY KEY (organization_id, managed_facility_type_id)`?

Reply:
I can do that